test/functionalTest/cleanFiles.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# This script remove all the crap files that testHarness or valgrind suite 
+# use to produce in some cases (e.g. failing tests)
+
+find -name *.out -exec rm {} \;
+find -name *.diff -exec rm {} \;
+find -name *.regexpect -exec rm {} \;
+find -name *.shell -exec rm {} \;
+find -name *.shell.stdout -exec rm {} \;
+find -name *.shell.stderr -exec rm {} \;
+find -name *.shellInit -exec rm {} \;
+find -name *.shellInit.stderr -exec rm {} \;
+find -name *.shellInit.stdout -exec rm {} \;
+find -name *.teardown -exec rm {} \;
+find -name *.teardown.stderr -exec rm {} \;
+find -name *.teardown.stdout -exec rm {} \;
+find -name *.log -exec rm {} \;
+find . -type d -empty -exec rmdir {} \;
```

Comment:
The typical usage case is:

```
cd test/functionaTests
./testHarness
(something bad happens, a lot of crap file appear under cases/ subdirectories)
./cleanFiles.sh
```

Using `find test/functionalTests/cases ...` would break that case, isn't it? Or maybe I'm not getting the point...